sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -45,11 +45,11 @@ statement
     | ALTER DATABASE identifier SET DBPROPERTIES tablePropertyList     #setDatabaseProperties
     | DROP DATABASE (IF EXISTS)? identifier (RESTRICT | CASCADE)?      #dropDatabase
     | createTableHeader ('(' colTypeList ')')? tableProvider
-        (OPTIONS tablePropertyList)?
+        (OPTIONS optionParameterList)?
         (PARTITIONED BY partitionColumnNames=identifierList)?
         bucketSpec?                                                    #createTableUsing
     | createTableHeader tableProvider
-        (OPTIONS tablePropertyList)?
```

Comment:
Why not generalize the `tableProperty` rule and use `optionValue` (rename it to something more consistent) as its value rule? Seems easier.